NorthstarDLL/mods/modmanager.cpp

```diff
@@ -460,7 +532,16 @@ void ModManager::LoadMods()
 			// behaviour is for defining same convar multiple times
 			if (!R2::g_pCVar->FindVar(convar->Name.c_str()))
 			{
-				new ConVar(convar->Name.c_str(), convar->DefaultValue.c_str(), convar->Flags, convar->HelpString.c_str());
+				new ConVar(
```

Comment:
maybe add a check if the convar actually has a callback defined?

Reply:
Is that needed? Imo there's no real benefit and it just clutters up the code